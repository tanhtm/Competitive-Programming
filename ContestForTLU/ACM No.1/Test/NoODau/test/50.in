692 162
1 1 6 7 8 8 9 10 10 11 20 20 20 21 23 24 27 29 30 30 36 42 43 43 47 48 49 49 50 52 53 54 54 55 56 59 60 60 61 63 64 64 70 72 73 74 76 76 78 79 79 80 86 88 88 89 90 91 92 94 95 97 99 100 100 101 106 110 111 112 113 116 118 119 120 124 125 130 130 132 134 134 134 135 136 136 136 141 142 143 144 145 146 148 150 152 152 153 154 154 155 161 167 171 173 177 180 181 182 184 187 191 192 193 194 194 194 197 197 206 207 211 213 214 215 216 217 217 219 219 219 221 221 221 221 222 222 224 224 225 228 228 230 231 233 234 235 237 238 240 240 240 242 244 244 245 248 253 255 257 257 263 264 264 265 268 273 274 276 278 279 279 279 280 281 284 284 284 285 285 285 285 286 290 292 293 293 293 296 296 298 300 302 303 303 304 308 309 315 316 316 316 318 321 321 323 323 326 327 328 328 331 334 335 335 335 336 341 343 346 347 350 354 364 365 365 367 368 368 369 372 372 375 377 378 380 380 381 381 385 386 389 389 390 392 395 396 396 399 400 400 401 403 406 407 408 411 412 412 415 416 418 418 418 418 418 420 420 422 424 424 425 427 428 429 429 431 432 432 433 435 436 436 437 437 437 439 443 444 444 444 446 446 447 448 450 450 453 453 457 459 459 461 461 461 462 463 463 463 464 465 465 467 468 469 471 471 471 476 476 478 480 482 483 484 484 484 484 485 485 486 487 488 495 496 498 499 501 503 504 507 508 508 510 513 515 516 516 517 518 520 523 528 529 530 530 531 536 538 538 538 544 544 545 546 547 548 550 550 550 550 553 556 557 558 558 559 559 561 561 561 563 565 565 566 569 570 570 572 572 573 574 576 577 579 580 580 580 582 587 589 592 594 596 597 603 604 606 608 608 609 609 611 611 613 614 615 618 618 618 618 618 619 619 619 620 620 621 622 625 626 627 629 630 631 633 635 638 641 642 643 646 647 649 651 652 657 661 662 662 663 663 664 668 668 669 669 669 670 674 674 675 676 677 679 679 685 688 689 690 691 694 694 694 695 696 696 697 697 698 699 699 700 700 703 703 705 706 706 707 710 710 710 712 712 714 715 715 717 718 723 724 724 730 730 731 734 734 734 735 735 735 735 736 744 744 744 746 748 749 751 751 752 754 758 759 761 761 761 765 766 766 767 767 771 771 773 773 774 774 775 775 777 778 779 781 783 784 786 788 790 791 791 793 794 795 796 799 800 800 801 801 803 809 810 811 813 814 817 817 817 821 822 834 838 844 847 849 849 851 851 853 854 854 856 856 858 861 867 867 870 871 871 874 876 880 885 885 888 889 889 890 892 892 892 892 893 894 895 897 897 898 900 900 903 903 904 905 907 908 912 912 913 913 913 914 915 916 918 919 922 922 922 923 924 925 926 926 927 928 928 929 929 929 935 935 936 939 940 942 942 944 944 945 946 946 950 950 951 954 955 956 957 957 960 961 961 961 963 964 968 970 970 976 979 979 980 981 982 983 986 986 986 987 987 988 988 994 996 996 997 999 
41 396 719 746 812 698 246 562 314 321 889 360 284 464 21 863 866 313 552 542 587 220 504 799 634 964 497 387 893 961 486 933 709 556 31 520 605 276 81 918 597 969 277 232 784 650 94 650 314 645 543 900 216 46 50 202 9 899 588 253 859 425 538 567 332 568 86 937 195 167 854 791 487 483 22 271 484 468 272 797 464 814 48 680 211 97 881 219 995 468 824 854 244 361 420 575 928 506 511 122 24 717 265 510 199 638 132 34 105 403 830 921 216 877 600 426 973 832 997 320 651 820 173 894 180 944 468 459 449 331 932 824 47 548 686 597 186 817 630 642 220 459 562 787 687 513 213 11 344 561 330 994 380 854 887 911 798 707 369 598 37 652 
