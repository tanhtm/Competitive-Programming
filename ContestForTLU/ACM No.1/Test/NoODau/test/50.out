22
247
501
518
567
480
157
382
199
204
600
224
176
310
14
589
589
199
372
362
400
132
340
558
437
670
336
242
607
666
331
645
491
373
21
351
408
169
53
629
405
672
170
145
548
445
60
445
199
442
362
613
126
25
29
120
7
613
401
158
588
272
359
386
213
386
53
648
118
103
583
552
332
324
15
167
325
314
167
558
310
568
26
467
122
62
597
129
689
314
574
583
154
224
267
393
640
341
345
76
16
499
165
344
120
438
80
21
67
253
574
631
126
596
406
273
674
574
691
204
445
572
105
608
107
652
314
301
296
212
645
574
25
367
468
405
111
569
434
440
132
301
382
550
468
345
123
10
220
379
212
688
236
583
599
621
558
490
230
406
22
446
